src/components/UserInfo.jsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
```

Comment:
Overall, the UserInfo card looks great. I think we can make it look even better by utilizing more consistent spacing around the elements. For example, the URL link under the user's name has a differing top and bottom margin, and this amount of spacing seems different to the spacing around the bio.

Reply:
Right, I started out with equal spacing between the major sections, but it didn't look quite as nice to me. We can revisit this later maybe after getting feedback from everyone?

Reply:
Sure thing, sounds good to me!